app/partials/wallet-welcome.jade

```diff
@@ -6,52 +6,49 @@ blocket-loading
     .quickstart
       #intro-header.container
         .row-fluid
-          .span4.col.hidden-phone
-            h6 New Users
-            p
-              | Creating a new bitcoin wallet takes literally a few seconds. You will be able to send and receive payments immediately.
           .span4.col
-            h6 Existing Users
-            p If you have previously created a My Wallet account click below to login.
+            h6(translate="EXISTING_USERS")
+            p(translate="UPGRADE_COPY")
           .span4.col.hidden-phone
-            h6 Help And Support
+            h6(translate="LEGACY_LOGIN") 
             p
-              | Please consult the FAQ for commonly asked questions. For any problems with the site, please contact us.
-        .row-fluid
+              | {{ 'LEGACY_COPY' | translate }}
           .span4.col.hidden-phone
+            h6(translate="HELP_SUPPORT")
             p
-              a.btn.btn-primary(ui-sref="public.signup") Start A New Wallet
+              | {{ 'SUPPORT_COPY' | translate }}
+        .row-fluid
           .span4.col
             p
-              a.btn.btn-primary(ui-sref="public.login-no-uid") Login Now
+              a.btn.btn-primary(ui-sref="public.login-no-uid" translate="LOGIN_NEW")
+          .span4.col.hidden-phone
+            p
+              a.btn.btn-muted(href="https://blockchain.info/wallet/login" translate="LOGIN_LEGACY")
```

Comment:
Can you replace that URL with `/wallet/login`? That makes testing on staging easier.